src/libasr/codegen/llvm_utils.h

```diff
@@ -128,6 +128,10 @@ namespace LFortran {
 
         public:
 
+            static bool enable_bounds_checking;
```

Comment:
Making this private would be better now.

Reply:
Done!